internal/object/object.go

```diff
@@ -199,3 +201,9 @@ func NewIngress(name, ns, backendName string, backendPort int) *networkingv1.Ing
 		},
 	}
 }
+
+// TagFromImage returns the tag from the provided image.
+func TagFromImage(image string) string {
+	parsed := strings.Split(image, ":")
```

Comment:
This works as long as we don't pass something like: `projectcontour/contour` which I think we'll be ok with since it's internally managed.

Reply:
@stevesloka I updated the func to return  "" if the provided image does not contain `:`.